test-jest/chart/AreaChart.spec.tsx

```diff
@@ -55,50 +55,37 @@ describe('AreaChart', () => {
     });
   });
 
-  test.skip('Renders customized active dot when activeDot is set to be a ReactElement', () => {
+  test.skip('Renders customized active dot when activeDot is set to be a ReactElement', async () => {
     // eslint-disable-next-line react/prop-types
-    const ActiveDot = ({ cx, cy }) => <circle cx={cx} cy={cy} r={10} className="customized-active-dot" />;
+    const ActiveDot = ({ cx, cy }) => (
+      <circle cx={cx} cy={cy} r={10} className="customized-active-dot">
+        <text>dot-test</text>
+      </circle>
+    );
     const { container } = render(
-      <AreaChart width={400} height={400} data={data}>
-        <Area activeDot={<ActiveDot />} type="monotone" dataKey="uv" stroke="#ff7300" fill="#ff7300" />
-        <Tooltip />
-      </AreaChart>,
+      <div role="main" style={{ width: '400px', height: '400px' }}>
+        <AreaChart width={400} height={400} data={data}>
+          <Area
+            activeDot={({ cx, cy }) => <ActiveDot cx={cx} cy={cy} />}
+            type="monotone"
+            dataKey="uv"
+            stroke="#ff7300"
+            fill="#ff7300"
+          />
+          <Tooltip />
+        </AreaChart>
+      </div>,
     );
 
-    fireEvent.mouseEnter(container, { clientX: 200, clientY: 200 });
-    // fireEvent.mouseEnter(container.querySelector('.recharts-curve'));
+    const area = container.querySelector('.recharts-area-area');
 
-    expect(container.querySelectorAll('.recharts-active-dot')).toHaveLength(1);
-  });
+    if (area !== null) {
+      await userEvent.hover(area);
+      screen.debug(area);
+    }
```

Comment:
Is the debug needed in the test?
I use it while writing tests, but always remove it in the end.

Reply:
I think since its in a skip its fine for now